docs/user_documentation/ref-expressions.md

```diff
@@ -0,0 +1,61 @@
+# Expressions
+
+## Introduction
+For some entity type attributes an expression can be set that determines whether or not a condition is true based on the 
+attribute values e.g. to determine whether values are valid. The expression format is based on the 
+[Magma JavaScript API](http://wiki.obiba.org/display/OPALDOC/Magma+Javascript+API).
+
+### Examples
+The following expression returns true when the value of attribute 'myAttributeName' of an entity only contains
+alphanumberic characters: 
+```js
+$('myStringAttributeName').matches(/^[a-z0-9]+$/i).value()
+```
+Is this example:
```

Comment:
Is this example: -> In this example:

Reply:
Agreed. Feel free to make these minor edits yourself.